calendar.html

```diff
@@ -2,219 +2,42 @@
-							}
-						}
-					});
-				});
-				
-				
-				
-				$('#calendar').fullCalendar({
-					defaultDate: new Date(),
-					editable: true,
-					eventLimit: true, // allow "more" link when too many events
-					selectable: false,
-					select: false,
-					googleCalendarApiKey: "AIzaSyBrJY4ayDg3Dy9Z8obYvYv2MEF_7sWBT10",
-					events: {
-						// replace this with your own google calendar id.
-						googleCalendarId: "j7q7ccdcdmmv6sa5iu7ps3d2ec@group.calendar.google.com"
-					},
-					
-					header: {
-						left: '',
-						center: 'title',
-						right: 'today prev,next'
-					},
-					
-					buttonIcons: {
-						prev: 'left-single-arrow',
-						next: 'right-single-arrow',
-					},
-					
-					views: {
-						month: { // name of view
-							titleFormat: 'MMMM YYYY',
-							weekMode: '5'
-							// other view-specific options here
-						}
-					},
-					
-					eventRender: function( event, element, view ) { 
-						if (event.title == "Monday Night Lindy Hop") {
-							//apply your logic here, make changes to element.
-							element.css('background', 'red');
-							element.css('border-color', 'red');
-							//element.find('.fc-event-title').append("moo");
-							}else if (event.title == "Event"){
-							element.css('background', 'blue');
-							element.css('border-color', 'blue');
-							//element.find('.fc-event-title').append("moo");
-						}
-						
-						var tooltipRender = ('<div>'+event.title+'</div>');
-						element.qtip({
-							content: tooltipRender,
-							style: { classes: 'font-size: 16px;'},
-							position: {
-								my: 'center left',
-								at: 'top center',
-								
-								target: $(element)
-							}
-						});
-					},
-					
-					eventClick: function(calEvent, jsEvent, view) {
-						var t = calEvent.title;
-						var startDate = calEvent.start + "";
-						var endDate = calEvent.end + "";
-						
-						var title = calEvent.title;
-						for (var i = 0; i < title.length; i++){
-							title = title.replace(" ", "_"); 
-						}
-						
-						var description = calEvent.description;
-						for (var i = 0; i < description.length; i++){
-							description = description.replace("\n", "_"); 	
-						}
-						
-						var location = calEvent.location + "";
-						for (var i = 0; i < location.length; i++){
-							location = location.replace(" ", "_"); 
-						}
-						
-						var hrefLink = 'iframe.html#' + title + "#" + description + "#" + startDate + "#" + endDate + "#" + location;
-						$.fancybox.open({
-							href : hrefLink,
-							type : 'iframe',
-							padding : 5
-						});
-						
-						
-					}
-					});
-				
-				$('#calendar').fullCalendar('renderEvent', calEvent)
-				
-			alert("what");
-			});
+		<meta charset="utf-8"/>
+        <meta http-equiv="content-type" content="text/html">
+        <meta name="viewport" content="width=device-width,user-scalable=no,initial-scale=1">
+		
+		<script src="scripts/vendor/jquery-1.10.1.min.js" type="text/javascript"></script>
```

Comment:
Also, have you considered using CDNJS or Bower? Local libraries (like what you're doing now) is most reliable, but also a lot of strain on your server. Not that this project is going to be very performance intensive. :p